src/ATA.Bluebook/ATA.Bluebook.Web/Common/HtmlExtensions/HtmlExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace ATA.Bluebook.Web.Common.HtmlExtensions
+{
+    public static class HtmlExtensions
+    {
+        public static string IsActive(this IHtmlHelper html, params string[] controllers)

```

Comment:
Rename the Method with "GetActiveRouteClassName" and update the Related References

Reply:
Done